docs/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -134,6 +134,37 @@ image. We recommend the [Debian image](https://hub.docker.com/_/debian/)
 since it’s very tightly controlled and kept minimal (currently under 150 mb),
 while still being a full distribution.
 
+### LABEL
+
+(Understanding Docker Object Labels)(../labels-custom-metadata.md)
+
+You can add labels to your image to help organize images by project, record
+licensing information, to aid in automation, or for other reasons. For each
+label, add a line beginning with `LABEL` and with one or more key-value pairs.
+The following examples show the different acceptable formats.
+
+```nohighlight
+# Set one or more individual labels
+LABEL com.example.version="0.0.1-beta"
+LABEL vendor=ACME\ Incorporated
```

Comment:
It's a bit weird that we use quotes where they are not strictly needed but not when the value contains a space. I know we are trying to show different ways here but I think we should only promote quoted versions if spaces are needed in value as they are much more readable.